src/redux/cities.js

```diff
@@ -0,0 +1,106 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// const CITIES_URL = 'https://countriesnow.space/api/v0.1/countries/cities';
+const API_KEY = 'a9f162e549927790acab08267ea7f9f2d6a44ac6';
+// const AIR_URL = 'http://api.waqi.info/feed/';
+// const CITY_LOCA = 'https://api.api-ninjas.com/v1/city?name=';
+// const LOCA_KEY = 'mloFqsSPluWQ8UWJ9pJM2w==xej9B1wbh7CIAo74';
```

Comment:
- Please remove all commented-out code, this would ensure your app is clean and easy to read and maintain. Please fix this 😄

Reply:
well resolved 👍🏾